### PR TITLE
Adds HTML and starter content

### PR DESCRIPTION
**Pull request by SL-Lawson**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Stephanie L Lawson - Portfolio</title>
+</head>
+<body>
+  <header>
+    
+    <Nav>
```
Comment (axamei) [inline about.html]: must be in lowercase. change to `<nav>`

File: about.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    <div>
+            <img src="images/minecraft-club-ppl.jpg" alt="Minecraft Club at PPL">
+    </div>                 
+  </main>
+  <footer>
+      <div> 
+        <p>&copy; 2019 Stephanie L Lawson</p>
+      </div>
+      <div> 
+        <button>
```
Comment (axamei) [inline about.html]: buttons are for actions. this should be a link to your resume.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    </div> 
+    <div>
+            <img src="images/minecraft-club-ppl.jpg" alt="Minecraft Club at PPL">
+    </div>                 
+  </main>
+  <footer>
+      <div> 
+        <p>&copy; 2019 Stephanie L Lawson</p>
+      </div>
+      <div>
```
Comment (axamei) [inline about.html:73]: missing closing `</div>`

File: contact.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Stephanie L Lawson - Portfolio</title>
+</head>
+<body>
+  <header>
+    
+    <Nav>
```
Comment (axamei) [inline contact.html]: should be lowercase.

File: contact.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+          <button> Submit </button>
+        </p>
+      </form>
+
+  </main>
+  <footer>
+      <div> 
+          <p>&copy; 2019 Stephanie L Lawson</p>
+        </div>
+        <div>
```
Comment (axamei) [inline contact.html:51]: missing closing `</div>`

File: index.html [hunk 6]
```diff
@@ -4,11 +4,58 @@
...
+      
+          </p> 
+    </div>
+
+  </main>
+  <footer>
+    <div> <p>&copy; 2019 Stephanie L Lawson</p>
+    </div>
+    <div> 
+      <button><a href="README.md">Read Me</a>
```
Comment (axamei) [inline index.html]: you don't have to include the readme file.

**Review by axamei — CHANGES_REQUESTED** [review]

**Review by SL-Lawson — COMMENTED** [review]
Oh, no! I didn't save them before submitting. I thought I had but I missed some.

File: about.html (hunk 2 above)
Comment (SL-Lawson) [reply about.html]: I will include an older resume as I am currently updating.

**Comment by axamei** [comment]
@SL-Lawson you can still make changes. Go ahead and add another commit to include the new changes.

**Comment by SL-Lawson** [comment]
I started to but ran out of time before I had to leave from work. I think I have it all fixed now and have figured out how to add more to the pull request. Thank you!
________________________________

**Review by axamei — APPROVED** [review]

**Pull request merged**